wgpu-types/src/lib.rs

```diff
@@ -988,6 +988,16 @@ bitflags::bitflags! {
         ///
         /// [VK_KHR_external_memory_win32]: https://registry.khronos.org/vulkan/specs/latest/man/html/VK_KHR_external_memory_win32.html
         const VULKAN_EXTERNAL_MEMORY_WIN32 = 1 << 63;
+
+        /// Enables R64Uint image atomic min and max.
+        ///
+        /// Supported platforms:
+        /// - Vulkan (with VK_EXT_shader_image_atomic_int64)
+        /// - DX12 (with SM 6.6+ emulated via Rg32Uint texture)
+        /// - Metal (with MSL 3.1+ emulated via RG32Uint texture)
```

Comment:
I wouldn't call this emulated

Reply:
what would you call it?

Reply:
```suggestion
        /// - DX12 (with SM 6.6+)
        /// - Metal (with MSL 3.1+)
```

Just completely omit it - it's not exposed to the user in any way.